Gems/Atom/Tools/ShaderManagementConsole/Code/Source/ShaderManagementConsoleApplication.h

```diff
@@ -113,6 +114,8 @@ namespace ShaderManagementConsole
 
         static void PyIdleWaitFrames(uint32_t frames);
 
+        AzToolsFramework::TraceLogger m_traceLogger;
```

Comment:
Can this be moved to the base AzQtApplication?  I see one being added here and in the MaterialEditorApplication?

Reply:
Discussed this offline, this was due to AzToolsFramework vs. AzQtComponents dependencies